clubs/admin.py

```diff
@@ -101,6 +102,20 @@ def club(self, obj):
         return obj.club.name
 
 
+class SubscribeAdmin(admin.ModelAdmin):
+    search_fields = ('person__username', 'person__email', 'club__name', 'club__pk')
+    list_display = ('person', 'club', 'email')
```

Comment:
The `email` field will always be empty, since it does not exist in the `Subscribe` model. Perhaps you meant to do `person__email`?